guacamole/src/main/java/org/apache/guacamole/rest/auth/AuthenticationService.java

```diff
@@ -208,6 +218,50 @@ private AuthenticatedUser updateAuthenticatedUser(AuthenticatedUser authenticate
     }
 
     /**
+     * Notify all bound AuthenticationSuccessListeners that a successful authentication
+     * has occurred. If any of the bound listeners returns false (indicating that the
+     * authentication should be rejected) a GuacamoleRejectedAuthenticationException is
+     * thrown.
+     *
+     * @param authenticatedUser
+     *      The user that was successfully authenticated
+     * @param session
+     *      Existing session for the user (if any)
+     * @throws GuacamoleException
+     *      If a filter throws an exception or if any filter rejects the authentication
+     */
+    private void notifyAuthenticationSuccessListeners(
+            AuthenticatedUser authenticatedUser, GuacamoleSession session)
+            throws GuacamoleException {
+        UserContext userContext = null;
```

Comment:
Style nit-pick: Probably good to add a space above this line between the header and body of the method.